build/deployments/docker-compose.yaml

```diff
@@ -1,5 +1,7 @@
 version: "3.8"
 
+name: pocket-v1
```

Comment:
There's only a gotcha for the first time `make compose_and_watch` is run after the change if previously ran without it.

`loki` is kind of a special container and basically the user has to destroy the previous docker-compose stack named `deployment`

Demo

https://user-images.githubusercontent.com/29378614/218596765-0950f606-5763-4032-9bb3-68fe3a30df37.mp4

We have a few options:
- I could catch the fact that loki is there from the previous version and remove it, it would leave us with some tech debt because at some point nobody will have that "dangling" setup anymore.
- We could write a note about it
- We could revert my change and stick to the IMHO ugly `deployments` setup.
- We could let users figure that out themselves

any other idea is welcome